### ISSUE TITLE
Add JAST USA game store library

### ISSUE THREAD

**Issue by bayazidbh** [Heroic-Games-Launcher/HeroicGamesLauncher#3589]
### Problem description

The JAST USA store is often the preferred store for many visual novel readers, due to having regional pricing and being DRM-free, but it has no native launcher of its own.

It would be great to have a simple access to it from Heroic for Linux Desktop and Steam Deck, both has been growing more popular for visual novel fans.

### Feature description

A library interface, similar to the current lineup and planned stores, for JAST USA downloading and running.

### Additional information

The store features 18+ games, therefore I'm comfortable with no Store menu for it (or for it to be hidden behind a toggle) and maybe even the Library feature being behind a toggle (which might be useful as the amount of supported Library grows).

**Comment by trivarialthea** [comment]
Also interested in this one or are you per chance accepting PRs for this? Not sure what the goal of heroic is and if you want to add any other launchers...

**Comment by Lariaa** [comment]
@trivarialthea yes, contributions are welcome and we are working on adding more store fronts in the future like EA based on maxima.

**Comment by Nyn69** [comment]
+1 Is the most used store for VNs after Steam. I wish we had it specially for an easier way to acces to them from the SDeck.

**Comment by arielj** [comment]
Closing, if any new store is added, it will happen independently of github issues. https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/4605
